package.json

```diff
@@ -17,15 +18,20 @@
     "url": "https://github.com/emote-chat/server/issues"
   },
   "dependencies": {
+    "bcryptjs": "^2.4.3",
     "bluebird": "^3.5.4",
     "body-parser": "^1.18.3",
     "dotenv": "^7.0.0",
     "express": "^4.16.4",
+    "express-jwt": "^5.3.1",
+    "jsonwebtoken": "^8.5.1",
     "path": "^0.12.7",
-    "pg": "^7.9.0",
     "pg-promise": "^8.6.5"
   },
   "devDependencies": {
-    "jshint": "^2.10.2"
+    "apidoc": "^0.17.7",
+    "fs": "0.0.1-security",
```

Comment:
`fs` is used to read the `init_db.sql` file for the tests run, so the test db can be wiped clean before running each test suite. We could probably just as easily have the query defined somewhere but I figured we might as well use it for testing purposes since we have it already written.